app/models/user.rb

```diff
@@ -16,6 +16,10 @@ class User < ActiveRecord::Base
   has_many :granted_policies, class_name: "Policy", foreign_key: "granter_id"
 
   after_create :grant_implicit_policy
+  
+  def timeout_in
+    5.minutes
```

Comment:
A timeout of 5 min can be a bit of a hassle, especially in development mode. I'd suggest moving this value to a setting in `settings.yml`, and keeping a much larger value in `development.yml`.